src/views/AfhandelingView.vue

```diff
@@ -39,52 +32,47 @@ import { ContactmomentAfhandelForm } from "@/features/contactmoment";
 import { useContactmomentStore } from "@/stores/contactmoment";
 import ZakenOverzicht from "@/features/zaaksysteem/ZakenOverzicht.vue";
 import { ref } from "vue";
-import type { ContactmomentObject } from "@/features/zaaksysteem/types";
 import { useZaaksysteemService } from "@/features/zaaksysteem/service";
 import { useContactmomentService } from "@/features/contactmoment";
 import type { Contactmoment } from "@/features/contactmoment/types";
 import SimpleSpinner from "@/components/SimpleSpinner.vue";
 import ApplicationMessage from "@/components/ApplicationMessage.vue";
+import { useToast } from "@/stores/toast";
+import { useRouter } from "vue-router";
 
-const contactmoment = useContactmomentStore();
+const { toast } = useToast();
+const router = useRouter();
+const contactmomentStore = useContactmomentStore();
 const saving = ref(false);
 const service = useZaaksysteemService();
 const contactmomentService = useContactmomentService();
 const errorMessage = ref("");
-const saved = ref(false);
 
-const zakenToevoegenAanContactmoment = (id: string) => {
-  contactmoment?.zaken.forEach((zaak) => {
-    const data = {
-      contactmoment:
-        window.contactmomentenBaseUri + "/objectcontactmomenten/" + id, //todo de hele url zou uit de response van het aanmaken contactmoment moeten komen
+const zakenToevoegenAanContactmoment = (contactMoment: { url: string }) => {
```

Comment:
is de aanpassing aan deze functie noodzakelijk voor deze story? 
deze fix betreft een feedback punt uit een eerdere story en is doorgeschoven als taak bij de klantbeeld story. als we het twee keer net anders gaan oplossen dan  wordt het een lastige merge.

Reply:
Niet echt. De window.contactmentenBaseUri werd hier nog gebruikt, maar die bestaat niet meer. Heb nu alles teruggedraaid behalve de bugfix.